_episodes/05-accelerating-lammps.md

```diff
@@ -0,0 +1,84 @@
+---
+title: "Introduction"
+teaching: 0
+exercises: 0
+questions:
+- "Key question (FIXME)"
+objectives:
+- "First learning objective. (FIXME)"
+keypoints:
+- "First key point. Brief Answer to questions. (FIXME)"
+---
+ # Accelerating LAMMPS using KOKKOS
+   
+## What is Kokkos?
+
+* supports OpenMP and GPU
+* provide excellent scalability to many OpenMP threads
+* 1-to-1 GPU to CPU support
+* Minimal data transfer between the host and the device
+* everything runs on GPU
+* supports modern GPUs only
+* supports double precision only
+
+## Important features of LAMMPS Kokkos package
+
+Kokkos can be used in conjunction with LAMMPS to accelerate its performance. This package helps to optimize LAMMPS performances for specific hardware architectures.
+But to obtain the Kokkos accelartion, LAMMPS needs to be built using Kokkos for those specific hardwares and it also requires propoer choices of input parametres to get an optimal performance.
+
+The list of LAMMPS features that is supported by Kokkos is given below:
```

Comment:
This list is likely to fall out-of-date quickly, instead maybe mention that a list of supported features exists and provide a reference. Put this in an information box